apps/services/bff/src/app/modules/proxy/proxy.service.ts

```diff
@@ -154,6 +154,12 @@ export class ProxyService {
         headers: {
           'Content-Type': reqHeaderContentType || 'application/json',
           Authorization: `Bearer ${accessToken}`,
+          ...(req.headers['user-agent'] && {
+            'user-agent': req.headers['user-agent'],
+          }),
+          ...(req.ip && {
+            'x-forwarded-for': req.ip,
```

Comment:
Let's forward the `x-forwarded-for` header if it is defined, like [we do elsewhere](https://github.com/island-is/island.is/blob/2814b986ae095fa97f6f8fb2923f0e76bbac8d1c/libs/auth-nest-tools/src/lib/jwt.strategy.ts#L69). In production the bff is behind a couple of CDNs / reverse proxies which add this header.